### PR TITLE
Bani- Barge in SIP Profile, 

### PR DESCRIPTION
**Pull request by Bani2791**
# 1. Barge Group in SIP Profile
https://bani-20924--connexcs-docs.netlify.app/setup/config/sip-profile/#steps-to-setup-the-sip-profile

## Steps to setup the SIP Profile

1. Go to **Setup :material-menu-right: Config :material-menu-right: SIP Profile** and you will see a screen as shown below. Then click the **blue `+` button**.

2. A window will appear after you click on the `+` button. It will let you create a new SIP Profile. Fill in the following details.

      + **Name**: This is a required field that will display your SIP Profile's name.
      + **Profile Capacity**: This field enables you to add the number of people who can use this profile to access the features.
      + **Flags**: This field allows you to select one or multiple features that the user can access.sip3
      + **Barge Group**: This field allows you add a group to the user. It allows the user to `listen` to the conversation of the specified group using Barge in [ConneXML-Barge Groups](https://docs.connexcs.com/class5/connexml/#dial).

!!! Note
    The values entered should be plain text, **^[a-z_0-9]+$ only**.

[3.Click](http://3.click/) on `Save` to create a new profile.

# 2. Tech Prefix in Routing

https://bani-20924--connexcs-docs.netlify.app/customer/routing/#tech-prefix

## Tech Prefix

**1. What is Tech Prefix?**
   
A Tech Prefix is a string of numbers, added to the dialed number (DNIS) before sending the call.

**2. Purpose of Tech Prefix:**

   + **Customer Identification**: It allows routing calls for different customers using the same inbound IP by specifying which customer the call belongs to.
   + **Routing Control**: It can be used to direct calls to specific routes or carriers depending on the prefix.

**3. How it Works?**

```mermaid
graph LR
A[Tech prefix added to the dialed number before sending the call] --> B{VoIP identifies the Tech Prefix}
B --> C{Strip Tech Prefix}
C --> D{Route Call Based on Rules configured for that Prefix}
```

**Example Workflow**

```mermaid
flowchart TB
    A[Call Ingress] -- A call is made to a number with a tech prefix (e.g., 123*123456789) ---B[Prefix Matching] -- The system checks if the prefix 123* matches a configured customer or route ---C[Stripping Prefix] -- The system strips off the 123*, leaving the actual number 123456789.---D[Call Routing] --Based on the tech prefix, the call is routed to the associated customer or carrier ---End
```

**4. Advantages**

+ Flexibility in routing calls to different customers using a shared IP or trunk.

### Use Case for the Tech Prefix

Using Tech Prefix with SIP User "Parameter Rewrites" allows for significant granularity to manage permissions for how to connect a user's calls.

1. Use Parameter Rewrite on the SIP User (found in **Customer :material-menu-right: Auth :material-menu-right: SIP User :material-menu-right: Parameter Rewrite**) to add a number for calls from this SIP User:

    

2. Add a Tech Prefix for that user in Routing. In this example, it would be 1234.
3. Set how you want those calls routed: Internal to Class5, out to a provider, etc.

!!! Example
    ```mermaid
    graph
    A[Incoming Call with 001*123456789] -- Call will be routed to Customer A --- B[Customer A Prefix: 001]
    subgraph Customers
    B[Customer A Prefix: 001] & C[Customer B Prefix: 002]
    end
    B[Customer A Prefix: 001]-- Prefix 001* will be stripped  before passing the call to the destination number 123456789 --- Destination
    ```
# 3. Package charge notification days
You can set the package notification date. An e-mail will be sent the number of days before you wish to recharge the package

https://bani-20924--connexcs-docs.netlify.app/setup/settings/options/

# 4. Default STIR Shaken Attestation (Customer > Edit) 

https://bani-20924--connexcs-docs.netlify.app/customer/customer/#__tabbed_1_2

+ **Default STIR Shaken Attestation**: The attestation level you select here is the same attestation level that will be applied to the CLIs that your customer adds from customer portal.

### CODE REVIEW

File: docs/setup/settings/options.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Configure **Options** to set key default behaviors for the system. Click **`Edit
     |**Payment Details**|You can add custom messages and display details such as bank account information, simplifying the payment process. This is for display purposes only.|
     |**Payment Description**|Description that's displayed for payment methods such as PayPal and Stripe.|
     |**Smart Loading of Grid Tables**|You can choose this option to automatically load the tables while you scroll the page.|
-    |**Package Charge Notification Days**|You can choose the number of days you wish to receive the notification for package charging.|
+    |**Package Charge Notification Days**|You can set the package notification date. An e-mail will be sent the number of days before you wish to recharge the package.|
```
Comment (AdamNawaz-connex) [inline docs/setup/settings/options.md:19]: You can set the package notification date, and an email will be sent the specified number of days before the package is scheduled to be recharged.

File: docs/setup/config/sip-profile.md [hunk 2]
```diff
@@ -12,14 +12,18 @@ When someone logs into the [Web Phone](https://docs.connexcs.com/webphone/), the
...
 
-2. A window will appear after you click on the `+` button. It will let you create a new SIP Profile. As you can see, the window has 3 fields for setting up the SIP Profile.![sip2](/setup/img/sip2.jpg)
+2. A window will appear after you click on the `+` button. It will let you create a new SIP Profile. Fill in the following details.
 
-    2.1 **Name**: This is a required field that will display your SIP Profile's name.
+      + **Name**: This is a required field that will display your SIP Profile's name.
+      + **Profile Capacity**: This field enables you to add the number of people who can use this profile to access the features.
+      + **Flags**: This field allows you to select one or multiple features that the user can access.![sip3](/setup/img/sip3.jpg)
+      + **Barge Group**: This field allows you add a group to the user. It allows the user to `listen` to the conversation of the specified group using Barge in [ConneXML-Barge Groups](https://docs.connexcs.com/class5/connexml/#dial).
```
Comment (AdamNawaz-connex) [inline docs/setup/config/sip-profile.md:22]: This feature allows you to add a group to a user. The user can then listen to conversations within the specified group using the Barge functionality in [ConneXML-Barge Groups].

File: docs/setup/settings/options.md (hunk 1 above)
Comment (Bani2791) [reply docs/setup/settings/options.md:19]: done

File: docs/setup/config/sip-profile.md (hunk 2 above)
Comment (Bani2791) [reply docs/setup/config/sip-profile.md:22]: done

**Review by AdamNawaz-connex — APPROVED** [review]

**Pull request merged**